src/Core/tests/Benchmarks/Benchmarks/ShellBenchmarker.cs

```diff
@@ -7,10 +7,12 @@ namespace Microsoft.Maui.Handlers.Benchmarks
 	[MemoryDiagnoser]
 	public class ShellBenchmarker
 	{
-		[Benchmark]
-		public async Task GoTo()
+		Shell shell;
+
+		[GlobalSetup]
+		public void Setup()
 		{
-			var shell = new Shell();
+			shell = new Shell();
```

Comment:
At the time I wrote this benchmark, I was actually trying to make `new Shell()` faster...

This change is fine, though. 👍

Reply:
If you want I can create another PR to benchmark it. I've been working on some changes on GoToAsync to improve perf. and refactoring this way will be good to have a better notion about the changes.